can-component.js

```diff
@@ -374,20 +374,16 @@ var Component = Construct.extend(
 				componentTagData.scope = new Scope(componentScope);
 			}
 
-			// Hook up any partials with which the component was instantiated
-			var componentPartials = componentTagData.partials;
-			if (componentPartials !== undefined) {
-				options.partials = {};
-				for (var name in componentPartials) {
-					var partial = componentPartials[name];
-
-					// Check if it’s already a renderer function or
-					// a string that needs to be parsed by stache
-					if (typeof partial === "function") {
-						options.partials[name] = partial;
-					} else if (typeof partial === "string") {
-						var debugName = string.capitalize( string.camelize(name) ) + "Template";
-						options.partials[name] = stache(debugName, partial);
+			// Hook up any templates with which the component was instantiated
+			var componentTemplates = componentTagData.templates;
+			if (componentTemplates !== undefined) {
+				for (var name in componentTemplates) {
```

Comment:
are there other for/in loops in this function?

Reply:
There aren’t; I’m going to take this as a vague suggestion to use `canReflect.eachKey` instead :P